transports/msmq/routing_content_core_[,6).partial.md

```diff
@@ -1,5 +1,8 @@
+
+NOTE: When using instance mapping, the settings will have no effect on **audit/error** queue and **publish/subscribe** (except for subscription messages, subscription messages will be sent to all listed instances)
```

Comment:
I think there is no instance mapping file in NSB < v6, so this note wouldn't make much sense here, does it?

Reply:
Good point, so keep the partial and have it for 6 and up?